apps/laboratory/tests/shared/pages/WalletPage.ts

```diff
@@ -36,12 +36,15 @@ export class WalletPage {
    */
   async handleSessionProposal(opts: SessionParams) {
     const variant = opts.accept ? `approve` : `reject`
+    await this.page.getByTestId(`session-${variant}-button`).isEnabled()
     await this.page.getByTestId(`session-${variant}-button`).focus()
     await this.page.keyboard.press('Space')
   }
 
   async handleRequest({ accept }: { accept: boolean }) {
     const variant = accept ? `approve` : `reject`
-    await this.page.getByTestId(`request-button-${variant}`).click()
+    await this.page.getByTestId(`session-${variant}-button`).isEnabled()
+    await this.page.getByTestId(`session-${variant}-button`).focus()
+    await this.page.keyboard.press('Space')
```

Comment:
Why `.press('Space')` instead of `.click()`?

Reply:
Took me a while to figure out. Will add a comment in the next iteration. But for whatever reason on Chromium it has to be this flavor and `click` doesn't work for this specific button. In Firefox it works. I can click myself.
Low prio to change this in react-wallet which is used here but managed in a different repo